pkg/redis/mocks/Client.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.13.1. DO NOT EDIT.
+// Code generated by mockery v2.15.0. DO NOT EDIT.
```

Comment:
@j4k4 what do you think, should we update mockery? If so, should we simply go with the most recent version (v2.16.0)?

Reply:
not sure about the impact and how much work that would be

Reply:
maybe @BastiJD can check that for us?

Reply:
i did a `go install github.com/vektra/mockery/v2@v2.16.0` and recreated all mocks with:
```shell
BASE=$PWD
for folder in $(grep -H -r "go:generate mockery" * | cut -d':' -s -f1 | grep "\\.go" | xargs -n1 dirname | uniq)                                                                   ─╯
      do
        echo "processing ${folder}"
        cd ${BASE}/${folder}

        # gather all go generate statements for mockery and cut away the parts that are not valid shell code
        if ! ls | grep ".go" > /dev/null; then
          continue
        fi
        grep -H "go:generate mockery" *.go | cut -d'/' -f3 | cut -b 13- | sh
      done

```
and then did a go test and it works just fine. So seems there is no breaking changes in gosoline

Reply:
To not mix changes too much I will regenerate the mocks with v2.13.1 and create a ticket to update mockery. I will pick it up in the next few days.